### DISCUSSION TITLE
Differences in results between FDS versions

### DISCUSSION THREAD

**Discussion by MNJOLE** [firemodels/fds#13090] [Q&A] [answered]
![버전에 따른 결과 해석 차이](https://github.com/firemodels/fds/assets/166811458/be8b4af0-abcf-4fb7-bbad-639dd7ad99a4)
![버전에 따른 결과 해석 차이_2](https://github.com/firemodels/fds/assets/166811458/e6524e6d-1b00-4720-a6a9-1e38405edbe4)

Hello, I am currently facing difficulties due to differences in results between versions of FDS.

The blue graph is from version 6.3.0, and the red graph is from version 6.9.1.

I have simulated a scenario where PMMA undergoes pyrolysis under a constant heat flux. However, I have noticed that the maximum HRR and the decreasing trend of HRR are interpreted differently between the versions. The same FDS code yields these different results.

I would like to understand what differences exist between the two versions. Additionally, I would appreciate any advice on the best approach to handle such discrepancies between versions.

Thank you always for your help.

**Comment by TristanHehnen** [comment, marked as answer]
There have been [many changes between 6.3.0 and 6.9.1](https://github.com/firemodels/fds/wiki/FDS-Release-Notes) and I would not expect them to produce the same results. I'm not sure if it is easy to pinpoint this one thing that is responsible. Also, the description of your setup is too thin to make meaningful assumption. For example, the default chemical formula of soot was recently changed to be pure carbon. With this, the heat of combustion changes. If your setup involves gas phase combustion this may be part of the reason. There have also be changes to the wall turbulence models and the solid phase in general.

I strongly recommend to run all simulations of a given project with the same FDS version, to keep consistency. If you have to change versions you may need to re-run simulations to achieve consistency again.
You can also have a look in the validation guides of both versions to see if the changes in the presented data is similar to what you observe.

**Reply by drjfloyd** [reply]
As Tristan noted it is likely that a number of the thousands of source code changes in the 9 years since 6.3 have impacted your specific simulation. We do try to capture major FDS changes in the FDS Release Notes found on the wiki tab on this webpage.
